project/plugins.sbt

```diff
@@ -1,9 +1,6 @@
-addSbtPlugin("com.typesafe.sbt"          % "sbt-git"                  % "1.0.0")
-addSbtPlugin("com.jsuereth"              % "sbt-pgp"                  % "2.0.1")
 addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat"             % "0.1.13")
 addSbtPlugin("org.lyranthe.sbt"          % "partial-unification"      % "1.1.2")
 addSbtPlugin("com.github.gseitz"         % "sbt-release"              % "1.0.13")
```

Comment:
Can we remove this. sbt-ci-release should have removed the need for this entirely.

Reply:
Done.